docs/build-repro.md

```diff
@@ -167,3 +167,50 @@ An example `policy.json`:
 ```
 
 Any source type is supported, but how to pin a source depends on the type.
+
+## `SOURCE_DATE_EPOCH`
+[`SOURCE_DATE_EPOCH`](https://reproducible-builds.org/docs/source-date-epoch/) is the convention for pinning timestamps to a specific value.
+
+The Dockerfile frontend supports consuming the `SOURCE_DATE_EPOCH` value as a special build arg, since Dockerfile 1.5 (BuildKit 0.11).
```

Comment:
Technically 1.5 just has a fallback. The feature is in buildkit.

I think this should say that support was added in `v0.11`, minimal support is also on older BuildKit when using Dockerfile 1.5 frontend.

Reply:
updated